### PR TITLE
Features/migrations

### PR DESCRIPTION
**Pull request by jciannuzzi**
These are the migration tables for clients and case information.

### CODE REVIEW

**Comment by rivyyriv** [comment]
Great Job, Jon!

The only thing I would consider is possibly changing the date fields to a string. Technically this is a date but it is more of an arbitrarily one. And dates are harder to work within JS. I could be wrong here.

File: package.json [hunk 1]
```diff
@@ -56,14 +56,15 @@
     "cookie-parser": "~1.4.4",
     "cors": "^2.8.5",
     "debug": "~2.6.9",
+    "docker": "^1.0.0",
```
Comment (ashtilawat) [inline package.json]: Do we need the docker dependency here? I would prefer maintaining the package.json we have already, but if this is necessary, we can keep it.

File: package.json [hunk 2]
```diff
@@ -56,14 +56,15 @@
...
     "dotenv": "^8.2.0",
     "express": "~4.16.1",
     "faker": "^4.1.0",
     "helmet": "^3.23.1",
     "http-errors": "~1.6.3",
-    "knex": "^0.21.1",
+    "knex": "^0.21.21",
     "morgan": "~1.9.1",
```
Comment (ashtilawat) [inline package.json:65]: Let's add that these dependencies were updated to your description

File: docker-compose.yml [hunk 3]
```diff
@@ -5,7 +5,7 @@ services:
     image: postgres
     container_name: pg-api-starter
     ports:
-      - '5400:5432'
```
Comment (ashtilawat) [inline docker-compose.yml]: Let's change this back to 5400 for the repo. I think we updated this for your PC only.

**Review by ashtilawat — CHANGES_REQUESTED** [review]
Let's change back the port to 5400. I'm thinking we don't need the docker dependency in the package. Let me know why it is important if you want to keep it.

File: package.json (hunk 1 above)
Comment (jciannuzzi) [reply package.json]: As far as I know I didn't touch the package.json file so I don't really know how that got in there.

**Review by ashtilawat — APPROVED** [review]

**Review by miketillman95 — APPROVED** [review]

**Comment by dujules23** [comment]
[As a case manager, I want to be able to manage a family’s case in one place, so that I don’t have to move across multiple systems.](https://trello.com/c/2EHqZvqr/3-as-a-case-manager-i-want-to-be-able-to-manage-a-familys-case-in-one-place-so-that-i-dont-have-to-move-across-multiple-systems)

**Pull request merged**